server/users.js

```diff
@@ -70,7 +70,7 @@ Meteor.methods({
     );
   },
   numberOfPostsToday: function(){
-    console.log(numberOfItemsInPast24Hours(Meteor.user(), Posts));
+    console.log(numberOfItemsInPast24Hours(Meteor.user(), Posts)); //does anyone know why is it here?!
```

Comment:
yea, this doesn't seem terribly useful. If this function still exists on master it's probably fine to delete it (make sure to delete it wherever it's called too).